csaf_2.0/json_schema/csaf_json_schema.json

```diff
@@ -262,10 +270,12 @@
           "type": "object",
           "properties": {
             "namespace": {
-              "type": "string"
+              "type": "string",
+              "format": "uri"
```

Comment:
I am so used to namespaces not being URIs that I wonder if we better leave the refined type to the producer and consumer?